tutorials/plot_ecog.py

```diff
@@ -45,3 +48,23 @@
 subjects_dir = mne.datasets.sample.data_path() + '/subjects'
 fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
 mlab.view(200, 70)
+
+###############################################################################
+# Sometimes it is useful to make a scatterplot for the current figure view.
+# This is best accomplished with matplotlib. We can capture an image of the
+# current mayavi view, along with the xy position of each electrode, with the
+# `snapshot_brain_montage` function.
+
+xy, im = snapshot_brain_montage(fig, mon)
+# Convert from a dictionary to array to plot
+xy_pts = np.vstack(xy[ch['ch_name']] for ch in info['chs'])
+
+# Define an arbitrary "activity" pattern for viz
+activity = np.linspace(100, 200, xy_pts.shape[0])
+
+# This allows us to use matplotlib to create arbitrary 2d scatterplots
+_, ax = plt.subplots(figsize=(10, 10))
+ax.imshow(im)
+ax.scatter(*xy_pts.T, c=activity, s=activity, cmap='coolwarm')
```

Comment:
my personal preference is to indicate activity only through color (as size may indicate distance in a 3d plot) but from your previous screenshots it seems that it is commonplace in ecog to use both color and size, right?